README.md

```diff
@@ -8,13 +8,73 @@
 + Delete a key-value pair from an object
 + Iterate over key-value pairs in an object
 
-
 ## What the Student Already Knows
 
 - Variables
 - Data Type- Strings and Integers
 - Functions
-- Booleans, Comparison Operators, Flow Control 
+- Booleans, Comparison Operators, Flow Control
 - Arrays
 
+##Understanding Objects
+
+JavaScript objects are a set of key-value pairs. Objects use the JavaScript object notation, for example, `{ 'key': 'value' }`. Each key inside an object may have any value type. For example, strings, integers, booleans, and functions. JavaScript does not care what the value is as long as it maps to a key in the set.
+
+To create an object, fire up a plain old REPL in Node.js. A REPL is a Read Eval Print Loop, and I invite you to type right along with me. Letʼs write this like so:
+
+```javascript
+> var o = { key1: 1 };
+```
+
```

Comment:
it's good that you show how the object looks but can you explain what each of the elements and the syntax in this are?